Jenkinsfile.meta

```diff
@@ -3,13 +3,14 @@ properties([
 	disableResume(),
 	durabilityHint('PERFORMANCE_OPTIMIZED'),
 	pipelineTriggers([
-		cron('@hourly'),
+		githubPush(),
+		cron('@daily'), // check periodically, just in case
```

Comment:
We have to move this back to `@hourly` for now so we can pick up completed builds (until we figure out / implement a better way to have those triggers make their way back into the system).